packages/headless/src/features/search/search-actions.ts

```diff
@@ -105,6 +105,14 @@ export interface ExecuteSearchThunkReturn {
   analyticsAction: SearchAction;
 }
 
+export interface PrepareForSearchWithQueryOptions {
```

Comment:
Unless I'm mistaken, this does not need to be exported ? Or does it affect the documentation ?

Reply:
Let's try
```suggestion
interface PrepareForSearchWithQueryOptions {
```